### PR TITLE
Document Tooling API compatibility matrix

### PR DESCRIPTION
**Pull request by donat**
image

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/reference/third_party_integration.adoc [hunk 1]
```diff
@@ -112,17 +112,24 @@ The main entry point to the Tooling API is the link:{javadocPath}/org/gradle/too
 [[sec:embedding_compatibility]]
 === Compatibility of Java and Gradle versions
 
-The Tooling API requires Java 8 or later.
-The Gradle version used by builds may impose <<compatibility.adoc#compatibility,additional Java version requirements>>.
+The following components should be considered when implementing Gradle integration: the Tooling API version, The JVM running the Tooling API client (i.e. the IDE process), the JVM running the Gradle daemon, and the Gradle version.
 
-The Tooling API supports running builds using Gradle 2.6 and later.
-Gradle 5.0 and up require clients to use Tooling API version 3.0 or later.
+The Tooling API itself is Java library published as part of the Gradle release.
```
Comment (ljacomet) [inline subprojects/docs/src/docs/userguide/reference/third_party_integration.adoc:117]: -> is a Java library

File: subprojects/docs/src/docs/userguide/reference/third_party_integration.adoc [hunk 2]
```diff
@@ -112,17 +112,24 @@ The main entry point to the Tooling API is the link:{javadocPath}/org/gradle/too
...
+Each Gradle release has a corresponding Tooling API version with a same version number.
```
Comment (ljacomet) [inline subprojects/docs/src/docs/userguide/reference/third_party_integration.adoc:118]: -> with the same

**Review by ljacomet — APPROVED** [review]
Two remarks, good to go after that.

File: subprojects/docs/src/docs/userguide/reference/third_party_integration.adoc (hunk 1 above)
Comment (donat) [reply subprojects/docs/src/docs/userguide/reference/third_party_integration.adoc]: Fixed.

File: subprojects/docs/src/docs/userguide/reference/third_party_integration.adoc (hunk 2 above)
Comment (donat) [reply subprojects/docs/src/docs/userguide/reference/third_party_integration.adoc]: Fixed

**Comment by donat** [comment]
@bot-gradle test and merge

**Pull request merged**